docs/source/settings.rst

```diff
@@ -826,7 +826,7 @@ statsd_prefix
 ~~~~~~~~~~~~~
 
 * ``--statsd-prefix STATSD_PREFIX``
-* ````
+* ``None``
```

Comment:
`docs/source/settings.rst` is automatically generated by `docs/gunicorn_ext.py` and it extracts all information from `gunicorn/config.py`. In this case, the default value of `statsd_prefix` is an empty string:

https://github.com/benoitc/gunicorn/blob/4c3ee6f4ec304722acb40d113fc4566f33bf3a6e/gunicorn/config.py#L1422

I think the best way to fix this would be to make `docs/gunicorn_ext.py` return `(empty string)` instead of \`\`\`\`.

Reply:
OK